src/utils/hooks/useLoadReview.tsx

```diff
@@ -1,34 +1,126 @@
 import { useEffect, useState } from 'react'
-import { useGetReviewQuery } from '../generated/graphql'
+import { ApolloError } from '@apollo/client'
+import {
+  ApplicationResponse,
+  ReviewResponse,
+  TemplateElement,
+  useGetReviewQuery,
+} from '../generated/graphql'
+import { ApplicationElementStates, SectionElementStates } from '../types'
+import useLoadApplication from './useLoadApplication'
+import useGetResponsesAndElementState from './useGetResponsesAndElementState'
+import getPageElements from '../helpers/getPageElements'
 
 interface UseLoadReviewProps {
   reviewId: number
+  serialNumber: string
 }
 
-const useLoadReview = ({ reviewId }: UseLoadReviewProps) => {
+const useLoadReview = ({ reviewId, serialNumber }: UseLoadReviewProps) => {
+  const [applicationName, setApplicatioName] = useState<string>('')
+  const [reviewSections, setReviewSections] = useState<SectionElementStates[]>()
   const [loading, setLoading] = useState(true)
   const [error, setError] = useState('')
 
+  const {
+    error: applicationError,
+    application,
+    templateSections,
+    isApplicationLoaded,
+  } = useLoadApplication({
+    serialNumber,
+  })
+
```

Comment:
Would be good if you could merge the changes from #219 into here (and adjust accordingly) so we can check the whole loading process works correctly. If it's not too much hassle?

Reply:
Oh, no problem. I can do that, just one moment.

Reply:
Done!